### DISCUSSION TITLE
Choose webcam device

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6541] [Q&A] [answered]
Hi, is it possible to choose a webcam device for qutebrowser? /dev/video0 is the default camera, but I'd like to use /dev/video2 which is a vritual camera created using v4l2loopback. How to tell qutebrowser which camera to use?

**Comment by toofar** [comment, marked as answer]
I don't think we have an API for it (see https://bugreports.qt.io/browse/QTBUG-70077). If the webpage doesn't give you an option to select then maybe you could try some silliness like `chmod +r /dev/video0` before opening the page and maybe the Qt code will skip over the ones it can't use?

**Reply by uwla** [reply]
I have another workaround: `mv /dev/video0 /dev/video0backup` then `ln -s /dev/video2 /dev/video0`. It basically createa symlink of the default webcam to the virtual webcam
